src/background/streams.ts

```diff
@@ -51,11 +52,14 @@ export class PortStreams extends Duplex {
 
   _write(chunk: any, encoding: string, callback: (error?: Error | null) => void): void {
     if (isRuntimeMessage(chunk)) {
-      const handle = (dst: string) => {
+      const handle = (dst: string, type) => {
```

Comment:
what's is type for `type`

Reply:
If the `isRuntimeMessage()` function can ensure type of `chunk` is RuntimeMessage, then the type of `type` is `InPageMsgTypes | BgMsgTypes | PopupMsgTypes`. But `isRuntimeMessage()` just return ture without any check. So I am not sure...

Reply:
I will add `InPageMsgTypes | BgMsgTypes | PopupMsgTypes` as the type of `type`.